Makefile

```diff
@@ -81,7 +81,7 @@ check-env-build:
 	fi
 
 requirements:
-	uv export --no-hashes --no-header --no-annotate --no-dev --format requirements.txt > requirements.txt
+	uv export --no-hashes --no-header --no-annotate --no-dev --format requirements-txt > requirements.txt
```

Comment:
Although it works, is there any reason to change `--format requirements.txt` to `--format requirements-txt` ?

Reply:
uv doc say requirements.txt https://docs.astral.sh/uv/reference/cli/#uv-export--format

Reply:
let me check if I can update the uv version